webapp/components/signup/components/signup_email.jsx

```diff
@@ -40,7 +40,7 @@ export default class SignupEmail extends React.Component {
         this.state = this.getInviteInfo();
     }
 
-    componentDidMount() {
+    static componentDidMount() {
```

Comment:
I did it in response to an eslint error. It claims that if a method doesn't use `this`, it should be made `static`, presumably because it has no need to modify the object's state. I guess that if you tried to reference `this` from within a `static` method, you'd get a compiler error of some kind.